src/Main.hx

```diff
@@ -227,10 +227,8 @@ class Main {
 				if (arg.charAt(0) == '-') {
 					throw 'Unknown argument "$arg"';
 				}
-				// remove ".d.ts" extension as this is a common point of confusion for users
-				var moduleName = if (arg.substr(arg.length - 5).toLowerCase() == '.d.ts') {
-					arg.substr(0, arg.length - 5);
-				} else arg;
+				// remove ".d.ts" extension as this is a common point of confusion for users, (remove extension for moduleName to check against module path lookup)
+				final moduleName = Path.withoutExtension(arg);
```

Comment:
Unfortunately this doesn’t quiet work, extensions are defined as characters after the last . whereas here we want to include the second to last .